src/olympia/access/acl.py

```diff
@@ -76,7 +78,10 @@ def check_collection_ownership(request, collection, require_owner=False):
     elif request.user.id == collection.author_id:
         return True
     elif not require_owner:
-        return collection.publishable_by(request.user)
+        return (
+            collection.pk == settings.COLLECTION_FEATURED_THEMES_ID and
+            action_allowed_user(
+                request.user, amo.permissions.COLLECTIONS_CONTRIBUTE))
     else:
         return False
```

Comment:
nit: I know this is just a minor change, and you didn't write the method... but I can't help feeling all these `if` and `elif`s could be rewritten - if not into a single return statement then something less redundant.

Reply:
I thought so as well, and I tried to change it to a single return statement but I found that it didn't really help... I didn't want to spend more time on it as it will eventually go away when we remove the legacy frontend.